tests/debugger_test.py

```diff
@@ -31,6 +33,11 @@
 
 config.parse_flags_with_absl()
 
+with contextlib.suppress(ImportError):
+  import pytest
+  pytestmark = pytest.mark.pjrt_c_api_unimplemented  # host callback
```

Comment:
Other question: does just declaring this in the sope actually mark tests within the file? I've always seen `pytest.mark` used as a decorator for a test function or test class.

Reply:
Looks great!